e2e-tests/cypress/tests/integration/playbooks/playbooks/edit/actions_spec.js

```diff
@@ -1183,58 +1028,71 @@ describe('playbooks > edit', {testIsolation: true}, () => {
+                    // * Verify no channel is selected
+                    cy.findByTestId('status-update-broadcast-channels').should(
+                        'have.text',
+                        'no channels',
+                    );
 
-                    // # Click on the toggle to enable the setting
-                    cy.get('#broadcast-channels').within(() => {
-                        cy.get('label input').click({force: true});
-                    });
+                    // # Open the broadcast channel widget
+                    cy.findByTestId('status-update-broadcast-channels').click();
 
-                    // # Open the broadcast channel widget and select a public channel
-                    cy.get('#playbook-automation-broadcast').
-                        click().
-                        type(testPrivateChannel.display_name + '{enter}', {delay: 200});
+                    // # select a private channel
+                    cy.get('#floating-ui-root').within(() => {
+                        cy.get('input').type(`${testPrivateChannel.display_name}{enter}{esc}`);
+                    });
 
-                    // # Save the playbook
-                    cy.findByTestId('save_playbook').click();
+                    // * Verify placeholder text is present
+                    cy.findByTestId('status-update-broadcast-channels').should(
+                        'have.text',
+                        '1 channel',
+                    );
 
                     // # Visit the selected playbook
-                    cy.visit('/playbooks/playbooks/' + testPlaybook.id + '/edit');
-
-                    // # Switch to Status updates tab
-                    cy.get('#root').findByText('Status updates').click();
+                    cy.visit('/playbooks/playbooks/' + testPlaybook.id + '/outline');
 
                     // * Verify placeholder text is present
-                    cy.get('#playbook-automation-broadcast').should(
+                    cy.findByTestId('status-update-broadcast-channels').should(
                         'have.text',
-                        testPrivateChannel.display_name,
+                        '1 channel',
                     );
+
+                    // # Open the broadcast channel widget
+                    cy.findByTestId('status-update-broadcast-channels').click();
+
+                    // * Verify channel name displayed
+                    cy.get('#floating-ui-root').within(() => {
+                        cy.findByText(testPrivateChannel.display_name).should('be.visible');
+                    });
                 });
 
                 it.skip('shows "Unknown channel" when private broadcast channel configured and user is not a member', () => {
                     // # Visit the selected playbook
-                    cy.visit('/playbooks/playbooks/' + testPlaybook.id + '/edit');
+                    cy.visit('/playbooks/playbooks/' + testPlaybook.id + '/outline');
```

Comment:
There are remaining 7 skipped tests. I wonder if that's intentional or going to address separately.

Reply:
Ah, thank you for clarifying the scope @saturninoabril. I was aiming to solve just the issues that were failing, and managed to un-skip a few that were now reliable, but didn't holistically evaluate all the skipped tests.

These tests have been skipped for about a year now, not because they were flaky, but because the implementation fundamentally changed without the tests being rewritten for the new screens. I'm inclined to simply rip them out vs. leaving them forever skipped, and reintroducing new coverage as needed in the future. Thoughts?

Reply:
Thanks Jesse and sounds good to me. I agree to remove for now and reintroduce as needed.